lib/util/git/index.ts

```diff
@@ -875,6 +877,16 @@ export async function prepareCommit({
           let contents: Buffer;
           // istanbul ignore else
           if (typeof file.contents === 'string') {
+            if (/migrate-config/g.exec(branchName)) {
+              const prettierConfigPath = readLocalPath(fileName);
+              const options = await prettier.resolveConfig(prettierConfigPath);
+              if (!is.null_(options)) {
+                file.contents = prettier.format(
+                  file.contents.replace(/\s{2}/g, ''),
```

Comment:
```suggestion
                  file.contents,
```
i think prettier will do it